internal/app/machined/main.go

```diff
@@ -114,7 +145,7 @@ func main() {
 	defer recovery()
 
 	// Subscribe to events.
-	init := EventBusObserver{&event.Embeddable{}}
+	init := EventBusObserver{&event.Embeddable{Chan: make(event.Channel, 1)}}
```

Comment:
I guess buffer of one still allows channel to "swallow" one event without feedback

Reply:
This isn't really needed I think. I can remove this since it will be addressed in the bus improvements we discussed.